server/src/helpers/translation.js

```diff
@@ -17,7 +17,9 @@ const detectLanguage = async text => {
     return data.detections[0][0].language
   } catch (error) {
     // eslint-disable-next-line no-console
-    console.error(error)
+    console.error(
+      "L'appel à Google Cloud  Translation API a échoué. Vérifiez les limites d'appels fixées pour ce projet"
```

Comment:
C'est important de logguer quand même l'erreur sous-jacente
```suggestion
      "L'appel à Google Cloud Translation API a échoué. Vérifiez les limites d'appels fixées pour ce projet.", error
```